tests/publiccloud/slem_basic.pm

```diff
@@ -71,9 +61,26 @@ sub run {
     $instance->run_ssh_command(cmd => 'systemctl is-enabled transactional-update.timer');
     $instance->run_ssh_command(cmd => 'systemctl is-enabled issue-add-ssh-keys');
 
-    # Ensure NetworkManager is used on SLEM 5.3+
-    my $expected_network_service = has_wicked() ? 'wicked' : 'NetworkManager';
-    $instance->ssh_assert_script_run("systemctl is-active $expected_network_service", fail_message => "$expected_network_service is not active");
+    # Test Networking. On SLEM6+ it needs to be NetworkManager. On <SLEM6 it must be either wicked or NetworkManager but never both at the same time
+    if (is_sle_micro('6.0+')) {
```

Comment:
I suggest to turn around the check for treating old versions as the exception, same as we should do by not checking for Tumbleweed but using that as reference. So how about `if (is_sle_micro('<6.0')) { …` (or corresponding syntax) so that eventually when older versions are EOL the code block can be removed. This also makes the code future-proof for any future product rebranding.

Reply:
Good idea!